tests/Doctrine/Tests/ORM/Query/SqlExpressionVisitorTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Query;
+
+use Doctrine\Common\Collections\ExpressionBuilder as CriteriaBuilder;
+use Doctrine\ORM\Mapping\ClassMetadata;
+use Doctrine\ORM\Persisters\Entity\BasicEntityPersister;
+use Doctrine\ORM\Persisters\SqlExpressionVisitor;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+use function method_exists;
+
+/**
+ * Test for QueryExpressionVisitor
```

Comment:
Looks wrong, and anyway, I don't think this is useful, please remove it.

Reply:
This is a correction from php-cbf, otherwise it's a php-cs violation... and I need it to know which version of `doctrine/collections` I'm using.

Reply:
Not having a comment is a phpcs violation? that would be a first…

Reply:
sorry, I thought you were talking about `use function method_exists;`

Reply:
It's fixed